apps/system/js/rocketbar.js

```diff
@@ -257,8 +257,10 @@
         case 'appforeground':
         case 'appopened':
         case 'utility-tray-overlayopening':
-          this.hideResults();
-          this.deactivate();
+          eventSafety(window, 'iac-search-results', function() {
```

Comment:
It does seem like it might work, but I think in this case 100ms could still be racey if things are just slow?